src/responses/yuzu.json

```
@@ -45,7 +45,12 @@
       "role": { "reply": "To claim your Patreon Discord role, please follow this guide: <https://support.patreon.com/hc/en-us/articles/212052266-Get-my-Discord-role>"},
       "release": { "reply": "yuzu builds can be manually downloaded on Github: <https://github.com/yuzu-emu/yuzu-mainline/releases>"},
-      "rec": { "reply": "For information on recommended settings and GPU drivers for yuzu, please refer to this page: <https://community.citra-emu.org/t/recommended-settings/319349>"}
+      "rec": { "reply": "For information on recommended settings and GPU drivers for yuzu, please refer to this page: <https://community.citra-emu.org/t/recommended-settings/319349>"},
+      "drivers": { "reply": "**Custom Drivers:** \nIf you have a device with a Qualcomm processor, you can explore the option of using custom drivers to potentially enhance performance and compatibility.\n\nPlease make sure to obtain the appropriate drivers for your specific device in the #android-drivers channel.\nFor Snapdragon 8 Gen1 or newer devices, you will need the Adreno 7xx drivers. For other devices, you will need the a6xx drivers."},
+      "ram": { "reply": "**Android RAM Requirements:** \nTo play most games on Yuzu for Android, a minimum of 8GB of RAM is currently required. While 6GB of RAM may work, it is only suitable for simple or small games"},
+      "powervr": { "reply": "**Unsupported PowerVR GPU** \nRegrettably, your device is equipped with a PowerVR GPU, which lacks the necessary features for us to provide support."},
+      "directories": { "reply": "Here are the important directories in the /Android/Data/org.yuzu-yuzu_emu folder: /nFirmware: /nand/system/Contents/registered (Installed with the button in the app. /nSaves: /nand/user/save/000000/profileID/gameID (You can directly import saves as zip files in the app, provided they are formatted correctly. This also works for importing saves from things like Skyline) /nMods: /load/gameID/ (No way to import mods through the app currently, you have to install them manually with a file manager) /nInstalled Updates/DLC: /nand/user/Contents/registered /n/nDue to android permission issues, most file managers cannot access the yuzu data directory.The easyest way to do so is to connect your phone to a pc and use its file manager. /nThere are also a number of 3rd party file manager apps which can access the directory."},
```

```suggestion
      "directories": { "reply": "Here are the important directories in the `/Android/Data/org.yuzu-yuzu_emu` folder: \n\nFirmware: `/nand/system/Contents/registered` \nInstalled using the button in the app. \n\nSaves: `/nand/user/save/000000/profileID/gameID` \nYou can directly import saves as .zip files in the app, provided they are formatted correctly. This also works for importing saves from emulators such as Skyline. \n\nMods: `/load/gameID/` \nThere is currently no way to import mods through the app, you must install them manually using a file manager. \n\nInstalled Updates/DLC: `/nand/user/Contents/registered` \n\nDue to Android permission issues, most file managers cannot access the yuzu data directory. The easiest way to do so is to connect your phone to a PC and use its file manager. \nThere are also a number of third party file manager apps which can access the directory."},
```

I've reformatted this entire section, it wasn't really reading right to me.

---

Yea that entire section was a mess, thanks for fixing it.